src/docfx/build/dependency/DependencyResolver.cs

```diff
@@ -41,26 +42,52 @@ internal class DependencyResolver
             return (error, content, child);
         }
 
-        public (Error error, string link, Document file) ResolveLink(SourceInfo<string> path, Document relativeTo, Document resultRelativeTo)
+        public (Error error, string link, Document file) ResolveRelativeLink(SourceInfo<string> path, Document relativeTo, Document resultRelativeTo)
```

Comment:
```suggestion
        public (Error error, string link, Document file) ResolveRelativeLink(SourceInfo<string> path, Document declaringFile, Document referencingFile)
```

Reply:
I've used `declaringFile` and `relativeToFile` to mimic `Path.GetRelativePath(string relativeTo, string path)`